pkg/controller/nodelifecycle/scheduler/taint_manager.go

```diff
@@ -40,9 +40,15 @@ import (
 )
 
 const (
-	nodeUpdateChannelSize = 10
-	podUpdateChannelSize  = 1
-	retries               = 5
+	// TODO (k82cn): Figure out a reasonable number of workers/channels and propagate
+	// the number of workers up making it a paramater of Run() function.
+
+	// NodeUpdateChannelSize defines the size of channel for node update events.
+	NodeUpdateChannelSize = 10
+	// UpdateWorkerSize defines the size of workers for node update or/and pod update.
+	UpdateWorkerSize     = 8
```

Comment:
should this be a function of the number of nodes in the system ?

Reply:
I don't think it's really needed. In small clusters there will be very little work to do so it doesn't hurt much to have more workers.
Ideally longer term it may depend on number of resource available for controller, but that's not needed now.